ec/src/lib.rs

```diff
@@ -313,6 +313,7 @@ pub fn prepare_g2<E: PairingEngine>(g: impl Into<E::G2Affine>) -> E::G2Prepared
 }
 
 /// A cycle of pairing-friendly elliptic curves.
+#[deprecated("Please use `PairingFriendlyCycle` instead")]
```

Comment:
oops, this should be

```suggestion
#[deprecated(note = "Please use `PairingFriendlyCycle` instead")]
```